standard-libraries/math.3.mr

```diff
@@ -0,0 +1,45 @@
+## Math functions for integers
```

Comment:
You are prefixing all the function names with `math-`. This should be replaced with namespaces. How about using Python's automatic namespace for modules?